src/create-render-context.js

```diff
@@ -69,8 +73,40 @@ class CustomResourceLoader extends ResourceLoader {
         return promiseToBuffer;
     }
 
+    _getFakeData(url: string, options: FetchOptions): ?Promise<Buffer> {
+        const ImageRegex = /^.*\.(jpe?g|png|gif)(?:\?.*)?/g;
+        const isImage = url.startsWith("data:image") || ImageRegex.test(url);
+        if (isImage) {
```

Comment:
Might be better to structure this as:

```
if (!isImage) {
    return CustomResourceLoader.EMPTY;
}

...
```

That way you don't have to have the majority of this function indented another level.